proto/src/determined/experiment/v1/experiment.proto

```diff
@@ -73,6 +73,8 @@ message Experiment {
   bool archived = 7;
   // The number of trials linked to the experiment.
   int32 num_trials = 8;
+  // The display name of the user that created the experiment.
+  string display_name = 18;
```

Comment:
If I understand this right, tasks needed to have a displayName on them to be sorted client-side,

If that's the case , I don't think that experiments need it?

Reply:
i think i added this because i thought the backend sorting needed it too. if adding it to `get_experiments.sql` is enough for the QueryProtof call to be sorted, I'll go ahead and remove.

Reply:
ok looks like it was necessary so keeping that change in